Makefile

```
@@ -151,6 +151,7 @@ endif
 	@echo "check the Catalyst demos"
 	MDD_BENCHMARK_PRECISION=1 \
 	$(ASAN_COMMAND) $(PYTHON) -m pytest demos/*.ipynb --nbmake $(PARALLELIZE)
+	$(ASAN_COMMAND) $(PYTHON) -m pytest demos/qml/*.ipynb --nbmake $(PARALLELIZE)
```

This is the command we also use elsewhere:
```suggestion
	$(ASAN_COMMAND) $(PYTHON) -m pytest demos --nbmake $(PARALLELIZE)
```